WIP-SQL/what-is-a-database-server.md

```diff
@@ -0,0 +1,61 @@
+# What is a database server?
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on database servers](http://ecomputernotes.com/fundamental/what-is-a-database/what-is-a-database-server)'
+
+---
+## Content
+
+A database server can be the *hardware* part of a database. A computer dedicated to perform database storage an retrieval. Such a hardware runs the database software  application (DBMS - database management system). It receives requests from the **client machines** and searches the collection for the **requested records**.
```

Comment:
misc fixes:
- storage *and* retrieval (not "an")
- "Such a hardware" -> "This computer"

"It receives requests from the **client machines** and searches the collection for the **requested records**." 
Change to:
The hardware is the location that the host for the database on the network is pointing to.